shell/common/platform_view.h

```diff
@@ -25,95 +25,507 @@ namespace flutter {
+  ///
+  ///             Embedders that wish to send platform messages to the framework
+  ///             may use the `DispatchPlatformMessage` method. This method is
+  ///             for messages that go the other way.
+  ///
+  /// @see        DisplatchPlatformMessage()
+  ///
+  /// @param[in]  message  The message
+  ///
+  virtual void HandlePlatformMessage(fml::RefPtr<PlatformMessage> message);
+
+  //----------------------------------------------------------------------------
+  /// @brief      Used by embedders to dispatch an accessibility action to a
+  ///             running isolate hosted by the engine.
+  ///
+  /// @param[in]  id      The identifier of the accessibility node on which to
+  ///                     perform the action.
+  /// @param[in]  action  The action
+  /// @param[in]  args    The arguments
+  ///
   void DispatchSemanticsAction(int32_t id,
                                SemanticsAction action,
                                std::vector<uint8_t> args);
 
+  //----------------------------------------------------------------------------
+  /// @brief      Used by embedder to notify the running isolate hosted by the
+  ///             engine on the UI thread that the accesibility tree needs to be
+  ///             generated.
+  ///
+  /// @attention  Subclasses may choose to override this method to perform
+  ///             platform specific functions. However, they must call the base
+  ///             class method at some point in their implementation.
+  ///
+  /// @param[in]  enabled  Whether the accessiblity tree needs to be generated.
+  ///
   virtual void SetSemanticsEnabled(bool enabled);
 
+  //----------------------------------------------------------------------------
+  /// @brief      Used by the embedder to sepcify the features to enable in the
+  ///             accessibility tree generated by the isolate. This information
+  ///             is forwarded to the root isolate hosted by the engine on the
+  ///             UI thread.
+  ///
+  ///             The engine does not care about the accesibilty feature flags
+  ///             as all it does is forward this information from the embedder
+  ///             to the framework. However, curious readers may refer to
+  ///             `AccessibilityFeatures` in `window.dart` for currently
+  ///             supported accessibility feature flags.
+  ///
+  /// @attention  Subclasses may choose to override this method to perform
+  ///             platform specific functions. However, they must call the base
+  ///             class method at some point in their implementation.
+  ///
+  /// @param[in]  flags  The fetures to enable in the accessibility tree.
+  ///
   virtual void SetAccessibilityFeatures(int32_t flags);
 
+  //----------------------------------------------------------------------------
+  /// @brief      Used by the framework to tell the embedder to apply the
+  ///             specified semantics node updates.
```

Comment:
The wording here feels unclear wrt the 'direction' the update is moving. Maybe something like:

Forwards updates to semantics nodes and custom accessibility actions actions to the platform-specific embedder. These updates originate in the framework.

or similar?

Reply:
Actually on re-reading, the directionality is clear as-is "Used by [the framework|embedders]" is sufficient.